CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 <!-- next version -->
 
+## 0.119.0
+
+### Components
+
+* [OpenTelemetry Collector - v0.119.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.119.0)
+* [OpenTelemetry Contrib - v0.119.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.119.0)
+* [Java auto-instrumentation - v1.33.6](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.33.6)
+* [.NET auto-instrumentation - v1.2.0](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/tag/v1.2.0)
+* [Node.JS - v0.53.0](https://github.com/open-telemetry/opentelemetry-js/releases/tag/experimental%2Fv0.53.0)
+* [Python - v0.50b0](https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/v0.50b0)
```

Comment:
Python needs to be updated https://github.com/open-telemetry/opentelemetry-operator/pull/3711

Reply:
in the versions.txt

Reply:
I will update the pr tonight :+1: